src/main/scala/zio/interop/guava/package.scala

```diff
@@ -106,12 +98,12 @@ package object guava {
               UIO.succeedNow(None)
           }
 
-        final def getRef[A](ref: FiberRef.Runtime[A])(implicit trace: ZTraceElement): UIO[A] = ref.get
+        final def getRef[A](ref: FiberRef[A])(implicit trace: ZTraceElement): UIO[A] = ref.get
```

Comment:
I'm not exactly sure whether this change is correct or not, as I'm a bit lost in FiberRef's changes between ZIO 2.0.0-RC2 and 2.0.0-RC5